src/common/models/robot.go

```diff
@@ -45,10 +46,10 @@ type RobotQuery struct {
 
 // RobotReq ...
 type RobotReq struct {
-	Name        string             `json:"name"`
-	Description string             `json:"description"`
-	Disabled    bool               `json:"disabled"`
-	Access      []*ResourceActions `json:"access"`
+	Name        string         `json:"name"`
+	Description string         `json:"description"`
+	Disabled    bool           `json:"disabled"`
+	Policy      []*rbac.Policy `json:"access"`
```

Comment:
We call it `policy` in the struct but call it `access` in the request body, any reason for this inconsistency?